examples/skills/pptx/scripts/office/pack.py

```
@@ -0,0 +1,160 @@
+from pathlib import Path
+
+import defusedxml.minidom
+
+from validators import DOCXSchemaValidator, PPTXSchemaValidator, RedliningValidator
+
+
+def pack(
+    input_directory: str,
+    output_file: str,
+    original_file: str | None = None,
+    validate: bool = True,
```

Ruff (0.15.13)

[warning] Boolean-typed positional argument in function definition

(FBT001)